.simplecov

```diff
@@ -0,0 +1,42 @@
+# RM_INFO is set when using Rubymine.  In Rubymine, starting SimpleCov is
+# controlled by running with coverage, so don't explicitly start coverage (and
+# therefore generate a report) when in Rubymine.  This _will_ generate a report
+# whenever `rake spec` is run.
+unless ENV['RM_INFO']
+  SimpleCov.start
+end
+
+SimpleCov.configure do
+  # ignore this file
+  add_filter '.simplecov'
+
+  # Rake tasks aren't tested with rspec
+  add_filter 'Rakefile'
+  add_filter 'lib/tasks'
+
+  #
+  # Changed Files in Git Group
+  # @see http://fredwu.me/post/35625566267/simplecov-test-coverage-for-changed-files-only
+  #
+
+  untracked = `git ls-files --exclude-standard --others`
+  unstaged = `git diff --name-only`
+  staged = `git diff --name-only --cached`
+  all = untracked + unstaged + staged
+  changed_filenames = all.split("\n")
+
+  add_group 'Changed' do |source_file|
+    changed_filenames.detect { |changed_filename|
```

Comment:
I'll make the change to `select`.  I use the [Jim Weirich](http://devblog.avdi.org/2011/07/26/the-procedurefunction-block-convention-in-ruby/) style for blocks: if a block has a yieldreturn, then I use `{ }` and if it's only run for side-effects, then I use `do end`.  If you're following the other style of `{ }` for one-liners (which I don't do anyway since you can't place a breakpoint on the inside of the block without breakpointing on the outside) and `do end` for multiline, then I'll change this to `do end` to conform to the project style.